traffic_portal/app/src/common/api/ServerService.js

```diff
@@ -140,7 +140,15 @@ var ServerService = function($http, locationUtils, messageModel, ENV) {
     };
 
     this.getDeliveryServiceServers = function(dsId) {
-        return $http.get(ENV.api['root'] + 'deliveryservices/' + dsId + '/servers').then(
+        return $http.get(ENV.api['root'] + 'servers?dsId=' + dsId).then(
```

Comment:
you could also pass this as a param instead of using direct query string construction, but it's less of a big deal since that's a number (we hope - JS be crazy)